site/pages/admin/user-management.page.tsx

```diff
@@ -113,7 +113,13 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
 
     const userList = userManagementSchema
         .parse(userRecords)
-        .filter((user) => user.organisation === sessionWithOrg.orgId);
+        .filter((user) => user.organisation === sessionWithOrg.orgId)
+        .sort((a, b) => {
+            return (
+                a.givenName.toLowerCase().localeCompare(b.givenName.toLowerCase()) ||
```

Comment:
I would just sort by email here since we don't display first and last names on the page

Reply:
done